tests/hook_main.py

```diff
@@ -2804,15 +2804,11 @@ def discount_delete(transaction):
 @hooks.before("Discount Codes > Get Discount Code Detail using the code > Get Discount Code Detail")
 def discount_code_get_detail_using_code(transaction):
     """
-    GET /discount-codes/DC101
+    GET event_id/1/discount-code/DC101
     :param transaction:
     :return:
     """
     with stash['app'].app_context():
-        event = EventFactoryBasic()
-        db.session.add(event)
-        db.session.commit()
```

Comment:
If you're going to remove this, how the discount code is going to be attached to the event?